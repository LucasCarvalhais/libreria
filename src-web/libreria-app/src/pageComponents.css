header {
    border: 5px solid black;
    background-image: linear-gradient(red 20%,  yellow 20%, yellow 80%, red 80%);
    font-family: 'Playfair Display', serif;
    font-size: 30px;
    color: #FFF;
    text-shadow: 3px 0px 5px darkred, 0px 3px 5px darkred, -3px 0px 5px darkred, 0px -3px 5px darkred;
    text-align: center;
}

nav {
    margin: 0px;
    padding: 0px;
    position: sticky;
    top: 0;
}

nav table {
    width: 100%;
    background-color: yellow;
}

nav table, td {
    border: 5px solid black;
    border-collapse: collapse;
}

nav td {
    font-size: 18px;
    text-align: center;
}

nav p {
    margin: 0px;
    padding: 10px;
}

nav td:hover {
    background-color: #CCCC00;
    text-decoration: underline;
    cursor: pointer;
}

footer {
    background-color: yellow;
    border: 5px solid black;
    margin: 0px;
    padding: 0px;
}

footer p {
    margin: auto;
    padding: 10px;
    font-size: 12px;
    text-align: center;
    width: 50%;
}