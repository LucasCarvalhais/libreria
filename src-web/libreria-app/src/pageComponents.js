import React from 'react';

const Header = () => 
    <header>
        <h1>Librería de Madrid</h1>
    </header>;

const Footer = () => 
    <footer>
        <p>
            En construcción... No es un sitio oficial, sólo un sitio para practicar el uso de Spring (back-end) y React (front-end). ¡Cualquier feedback or sugerencias son muy bienvenidas! :D
            </p>
    </footer>;

export {
    Header,
    Footer
}