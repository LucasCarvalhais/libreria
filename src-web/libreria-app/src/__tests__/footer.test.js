import React from 'react';
import renderer from 'react-test-renderer';
import { Footer } from '../pageComponents';

test('footer has a valid snapshot', () => {
    const component = renderer.create(<Footer />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
