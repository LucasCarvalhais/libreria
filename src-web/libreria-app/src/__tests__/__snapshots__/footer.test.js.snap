// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer has a valid snapshot 1`] = `
<footer>
  <p>
    En construcción... No es un sitio oficial, sólo un sitio para practicar el uso de Spring (back-end) y React (front-end). ¡Cualquier feedback or sugerencias son muy bienvenidas! :D
  </p>
</footer>
`;
